src/pages/components/button/PrimaryButton.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+
+const PrimaryButton = ({ onClick, className, title }) => {
+  return (
+    <button
+      className={`${className} px-8 py-2 bg-orange-900 rounded-lg text-white text-center font-semibold ml-0`}
```

Comment:
Add custom css for button instead of a component
```
//button.css
.primary-button {
  @apply px-8 py-2 bg-orange-900 rounded-lg text-white text-center font-semibold
}

//CookiePolicy.js
 <button
   className='primary-button ml-0 md:ml-8 mt-4 md:mt-0'
   onClick={onClickNoted}
 >
          Got it!
</button>
```